shell/browser/ui/file_dialog_win.cc

```diff
@@ -56,8 +56,11 @@ void ConvertFilters(const Filters& filters,
     spec.pszName = buffer->back().c_str();
 
     std::vector<std::string> extensions(filter.second);
+    if (extensions.empty())
```

Comment:
For the empty case, it seems redundant to create a vector with defined input to convert it back again into `LPCWSTR`. Can we just directory fill in `spec.pszSpec` for the empty case and skip creating both containers, the loop ?